src/mbgl/tile/geometry_tile_worker.cpp

```diff
@@ -404,6 +410,17 @@ void GeometryTileWorker::parse() {
     requestNewImages(imageDependencies);
 
     performSymbolLayout();
+#if MBGL_TILE_TIMING
+    milliseconds parseEndAt = duration_cast< milliseconds >(system_clock::now().time_since_epoch());
+    Log::Debug(Event::HttpRequest,
+               "Souce ID:%s, Canonical:%d/%d/%d, Requesting time:%llims",
```

Comment:
This is not a new problem, and it could be a separate PR, but I think it would be better if we got rid of:

`Log::record(EventSeverity severity, Event event, const char* format, ...)`

...and replaced it with something that's either templated based on the format args or exposes something like `std::stringstream` (or just require people to use the `std::string& msg` version of the method and build the string on their own). The problem is that `vsnprintf` ignores all type information in the format arguments, and it's really easy to write a security vulnerability by passing in an argument that doesn't match the format string. In debug-only timing code, I think i'm OK with using it, but I wouldn't want to have a `printf` in production code.